aws/iac/ebs.rego

```diff
@@ -11,7 +11,7 @@ default ebs_encrypt = null
 aws_issue["ebs_encrypt"] {
     resource := input.Resources[i]
     lower(resource.Type) == "aws::ec2::volume"
-    not resource.Properties.Encrypted
+    resource.Properties.Encrypted != true
```

Comment:
This is not covering Encrypted is not defined or "bool value as string" scenario. please refer [cloudtrail.rego](https://github.com/prancer-io/prancer-compliance-test/blob/master/aws/iac/cloudtrail.rego#L9).

Reply:
Fix: `git show e1d9096`

Reply:
added string check, fixed